### DISCUSSION TITLE
upgrade.sh fails 2.10.3

### DISCUSSION THREAD

**Discussion by makel05** [netbox-community/netbox#5663] [Q&A] [answered]
Any idea how I can resolve this issue? The upgrade.sh completes the database migration without issue, but ultimately the shell script fails with the following error.

`Checking for missing cable paths (python3 netbox/manage.py trace_paths --no-input)...
Retracing 3 cabled circuit terminations...
Side Z 5
Traceback (most recent call last):
  File "/opt/netbox-2.10.3/venv/lib64/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "dcim_cablepath_origin_type_id_origin_id_41b6f814_uniq"
DETAIL:  Key (origin_type_id, origin_id)=(7, 5) already exists.
`

**Comment by oletos7j** [comment]
I am also having this exact issue.

**Comment by jeremystretch** [comment, marked as answer]
Looks like you have a duplicate cable connection in your database somehow. Try running this in the shell (`manage.py nbshell`):

```
cp=CablePath.objects.get(origin_type_id=7, origin_id=5)
cp.origin
cp.origin.parent
```

That should point you to the offending termination object.
